v2/tracing_test.go

```diff
@@ -68,6 +69,14 @@ func Test_NewTracingHandler(t *testing.T) {
 			hasTraceContextOnContext: true,
 			hasTraceContextOnMessage: true,
 		},
+		{
+			name:                     "has traceCtx on message - extract from message- custom span name",
+			hasTraceContextOnContext: true,
+			hasTraceContextOnMessage: true,
+			spanEnrichFunc: func(c context.Context, t trace.Span) {
+				t.SetName("custom-span-name")
+			},
+		},
```

Comment:
Not sure how to test that span name has been set, or how we can test our enrichment in general. Maybe we should use a `ReadWriteSpan` instead...

Reply:
cmt outdated